lib/nostrum/cache/channel_cache.ex

```diff
@@ -24,8 +23,9 @@ defmodule Nostrum.Cache.ChannelCache do
   @doc ~S"""
   Retrieves a channel from the cache.
 
-  Internally, the ChannelCache process only stores DMChannel references. To get
-  channel information, a call is made to a `Nostrum.Cache.Guild.GuildServer`.
+  Internally, the ChannelCache process only stores
```

Comment:
Should probably link to channel struct's `dm_channel` type.

Reply:
It's there. It's below the line you linked.

Reply:
Don't know how I missed that. 👍